test/nbrowser/VersionUpdateBanner.ts

```diff
@@ -0,0 +1,69 @@
+import { version as installedVersion } from "app/common/version";
+import * as testUtils from 'test/server/testUtils';
+import * as gu from 'test/nbrowser/gristUtils';
+import {server, setupTestSuite} from 'test/nbrowser/testUtils';
+import {FakeUpdateServer, startFakeUpdateServer} from 'test/server/customUtil';
+import {assert, driver} from 'mocha-webdriver';
+
+describe('VersionUpdateBanner', function() {
+  this.timeout(300000);
+  setupTestSuite();
+
+  let oldEnv: testUtils.EnvironmentSnapshot;
+  let session: gu.Session;
+  let fakeServer: FakeUpdateServer;
+
+  afterEach(() => gu.checkForErrors());
+
+  before(async function() {
+    oldEnv = new testUtils.EnvironmentSnapshot();
+    process.env.GRIST_ALLOW_AUTOMATIC_VERSION_CHECKING = 'true';
+    process.env.GRIST_TEST_IMMEDIATE_VERSION_CHECK = 'true';
+    process.env.GRIST_DEFAULT_EMAIL = gu.session().email;
+    fakeServer = await startFakeUpdateServer();
+    process.env.GRIST_TEST_VERSION_CHECK_URL = `${fakeServer.url()}/version`;
+    await server.restart(true);
+  });
+
+  after(async function() {
+    await fakeServer.close();
+    oldEnv.restore();
+    await server.restart(true);
+  });
+
+  it('should not be shown to non-managers', async () => {
+    session = await gu.session().user('user2').personalSite.login();
+    await session.loadDocMenu('/');
+    assert.equal((await driver.findAll('.test-version-update-banner-text')).length, 0);
```

Comment:
Can you add some test id to a parent container and wait it to show up first?
```diff
--- a/core/app/client/ui/PagePanels.ts
+++ b/core/app/client/ui/PagePanels.ts
@@ -115,7 +115,7 @@ export function pagePanels(page: PageContents) {
     dom.autoDispose(sub2),
     dom.autoDispose(commandsGroup),
     dom.autoDispose(leftOverlap),
-    dom('div', page.contentTop, elem => { contentTopDom = elem; }),
+    dom('div', testId('top-panel'), page.contentTop, elem => { contentTopDom = elem; }),
     dom.maybe(page.banner, () => {
```

I think that some tests here are executed before the panels are even rendered.